components/raftstore-v2/src/router/message.rs

```diff
@@ -129,6 +130,8 @@ pub enum PeerMsg {
     SnapshotGenerated(Box<RaftSnapshot>),
     /// Start the FSM.
     Start,
+    /// Messages from peer to peer in the same store
+    AcrossPeerMsg(AcrossPeerMsg),
```

Comment:
Meaningless name, it should be specific.

Reply:
Done